src/labels/CustomLabels.labels

```diff
@@ -3622,6 +3622,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <protected>true</protected>
         <shortDescription>conMergeBtnLabel</shortDescription>
         <value>Merge</value>
+    </labels>
+     <labels>
+        <fullName>conMergeErrorNoDeleteObjPermission</fullName>
+        <categories>Contact Merge</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>conMergeErrorNoDeleteObjPermission</shortDescription>
+        <value>You must have Delete permissions on Contacts in order to merge Contacts.</value>
+    </labels>
+     <labels>
+        <fullName>conMergeErrorNoDeleteRecAccess</fullName>
+        <categories>Contact Merge</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>conMergeErrorNoDeleteRecAccess</shortDescription>
+        <value>You don't have the record sharing permissions to merge these Contacts. Your system administrator may need to adjust organization-wide sharing settings, sharing rules, or user roles.</value>
```

Comment:
Instead of using the apostrophe, you can escape this as `&apos;`?
ex: "You don&apos;t have the record sharing permissions ..."